src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/OffenceDetails.tsx

```diff
@@ -256,7 +256,21 @@ export const OffenceDetails = ({
           </div>
         </>
       )}
-      <BackToAllOffencesLink onClick={() => onBackToAllOffences()} />
+      <GridRow>
+        <GridCol>
+          <BackToAllOffencesLink onClick={() => onBackToAllOffences()} />
+        </GridCol>
+        <GridCol className={classes.button}>
+          {selectedOffenceIndex !== 1 && (
+            <SecondaryButton onClick={() => onPreviousClick()}>{"Previous offence"}</SecondaryButton>
+          )}
+          {selectedOffenceIndex !== offencesCount && (
+            <SecondaryButton className={classes.nextButton} onClick={() => onNextClick()}>
+              {"Next offence"}
+            </SecondaryButton>
+          )}
+        </GridCol>
+      </GridRow>
```

Comment:
is this identical to 149-160, maybe we could remove the duplication?

Reply:
I've pushed a change to extract it into a new component.